MANIFEST.in

```diff
@@ -7,15 +6,6 @@ include pyproj/*.pyd
 include pyproj/*.pyx
 include pyproj/*.pxd
 include pyproj/*.pxi
-include test/test.py
-include test/test2.py
 include test/sample.out
-include test/datum_shift.py
-include test/test_transform.py
-include test/geodtest.py
-include test/test_datum.py
 recursive-include docs *
-recursive-include unittest *
```

Comment:
I can't remember if there was a reason off the top of my head. I'll run some tests and get back.

Reply:
I'm not in favour of adding it back because I think it may then need:
```
prune */__pycache__/
```
to remove `test/__pycache__/` if it exists (maybe `prune test/__pycache__/`) and if this doesn't exist (normal situation) it generates a message:
```
no previously-included directories found matching '*/__pycache__/'
```
(I did have a `test/__pycache__/`, but after removing it I can't see what was creating it).

My rationale was no need to include things which are in the distutils standard definition including:
* anything that looks like a test script: test/test*.py
* Any of the standard README files (README, README.txt, or README.rst), setup.py (or whatever you called your setup script), and setup.cfg.

https://docs.python.org/3/distutils/sourcedist.html#specifying-the-files-to-distribute

README.md was different because although it is included automatically by setuptools it isn't in the disutils list above.

Reply:
Thanks for the clarification and reasoning, I agree with you on this decision. :+1: